modules/ting_search/ting_search.module

```diff
@@ -498,6 +520,23 @@ function ting_search_search_execute($keys = NULL, $conditions = NULL) {
   return $results;
 }
 
+/**
+ * Retrive the result for a ting_search query performed in this request.
+ *
+ * @param \Ting\Search\TingSearchResultInterface $new_search_result
+ *   New search result to make globally accessible.
+ *
+ * @return \Ting\Search\TingSearchResultInterface|NULL
+ *   The result or NULL if it is not available.
+ */
+function ting_search_current_results(TingSearchResultInterface $new_search_result = NULL) {
```

Comment:
This function should be use in more place. Search and replace 👍

Reply:
:+1: I could only find a single other place this is used.

Reply:
👍